main/zoe/guide/contracts/simple-exchange.md

```diff
@@ -2,116 +2,135 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 4/22/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 9/12/2020)
 ##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
 
-::: tip Out-of-date status
-Zoe's master branch is currently an Alpha release candidate. This doc
-and its underlying contract are in the process of being updated, and should be current with the release candidate in another few days. What you see here is out of date. We apologize for any inconvenience this may cause.
-:::
+The "simple exchange" is a very basic, un-optimized exchange. The simple
+exchange has an order book for one asset, priced in a second asset. The order
+book is naively kept as an array that is iterated over to look for matches
+every time a new order arrives.
 
-The "simple exchange" is a very basic, un-optimized exchange. The
-simple exchange has an order book for one asset, priced in a second
-asset. The order book is naively kept as an array that is iterated over
-to look for matches every time a new order arrives.
+## Contract API
 
-## Offer rules
+SimpleExchange is an exchange with a simple matching algorithm, which allows
+an unlimited number of parties to create new orders or accept existing
+orders. The notifier allows callers to find the current list of orders.
 
-The exchange accepts buy and sell limit orders only.
+The SimpleExchange uses Asset and Price as its keywords. The contract treats
+the two keywords symmetrically. New offers can be created and existing offers
+can be accepted in either direction.
 
-The Asset is treated as an exact amount to be exchanged, while the Price is a limit that may be improved on. This simple exchange does not support partial fills of orders.
+```js
+{ give: { Asset: simoleans(5) }, want: { Price: quatloos(3) } }
+{ give: { Price: quatloos(8) }, want: { Asset: simoleans(3) } }
+```
+
+The `want` term is treated as an exact amount to be exchanged, while the
+`give` term is a limit that may be improved on. This simple exchange does not
+partially fill orders.
+
```

Comment:
Is this true, or is it that the Price is improved upon where as the Asset is exact?

Reply:
That's what the previous version said, but the contract uses `swap()`, which takes each parties want from the other party.